lib/algo/NonuniformDMD.h

```diff
@@ -2,8 +2,6 @@
  *
  * Copyright (c) 2013-2022, Lawrence Livermore National Security, LLC
  * and other libROM project developers. See the top-level COPYRIGHT
- * file for details.
```

Comment:
Were these 2 lines intended to be deleted?

Reply:
Thanks for pointing that out. It was an accident.